tests/e2e/README.md

```diff
@@ -57,44 +57,46 @@ packages. `x/transfer/virtuous.go` defines X-Chain transfer tests,
 labeled with `x`, which can be selected by `./tests/e2e/e2e.test
 --ginkgo.label-filter "x"`.
 
-## Testing against a persistent network
+## Testing against an existing network
```

Comment:
I see in cortina 15 they are using persistent term and existing is introduced afterwards.